src/libsync/networkjobs.cpp

```diff
@@ -477,7 +477,8 @@ bool CheckServerJob::finished()
         if( status.contains("installed")
                 && status.contains("version")
                 && status.contains("versionstring") ) {
-
+            // Newer servers don't disclose anything in that empty string "version" anymore
```

Comment:
So then we will show instanceNotFound and we can't connect.  This needs to be changed as well!